src/harness/testcases/WINNF_FT_S_SDS_testcase.py

```diff
@@ -195,6 +195,31 @@ def test_WINNF_FT_S_SDS_10(self,config_filename):
       # Check Registration Response
       self.assertEqual(response[0]['response']['responseCode'], 104)
 
+  def generate_SDS_11_default_config(self, filename):
+    """Generates the WinnForum configuration for SDS_11. """
+    # Create the configuration for blacklisted domain proxy cert/key path
+
+    config = {
+      'domainProxyCert': self.getCertFilename("blacklisted_domain_proxy.cert"),
+      'domainProxyKey': self.getCertFilename("blacklisted_domain_proxy.key")
+    }
+    writeConfig(filename, config)
+
+  @configurable_testcase(generate_SDS_11_default_config)
+  def test_WINNF_FT_S_SDS_11(self, config_filename):
+    """Blacklisted certificate presented during registration..
+       Checks that SAS UUT response with fatal alert message.
```

Comment:
The second line of a docstring should be blank (and the third not indented):
https://www.python.org/dev/peps/pep-0008/#documentation-strings